setup.py

```diff
@@ -71,6 +71,8 @@
     ":sys_platform=='win32'": ['pyreadline'],
     # Python 3.4 and earlier require contextlib2 for temporarily redirecting stderr and stdout
     ":python_version<'3.5'": ['contextlib2'],
+    # Python 3.3 and earlier require enum34 backport of enum module from Python 3.4
+    ":python_version<'3.4'": ['enum34'],
```

Comment:
I added a conditional dependency on the enum34 module for Python 2.7.  This change here (and the associated one in the **tox.ini** file should fix the unit test failures on Python 2.7.